src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="field">
+          How old are You?
+          <input
+            type="number"
+            name="old"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="field">
+          Full date of birth:
+          <input
+            type="date"
+            name="fullDate"
+          >
+        </label>
+        <label class="field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="accept"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>Registration</legend>
+
+        <label>
+            E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="field">
+            Password:
+          <input
+            class="pass"
+            type="password"
+            name="pass"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+         Do you love cats?
+        <label>
+          <input
+            type="radio"
+            name="yes"
+            value="yes"
+          >
+            Yes
```

Comment:
I can choose both, fix it
<img width="231" alt="image" src="https://user-images.githubusercontent.com/102411130/220053475-6c233515-3343-40ea-98d0-a32896c83e14.png">